test/e2e/operator_test.go

```diff
@@ -161,6 +162,16 @@ func TestDefaultIngressControllerSteadyConditions(t *testing.T) {
 	}
 }
 
+// TestDefaultIngressClass verifies that the ingressclass controller has created
+// an ingressclass for the default ingresscontroller.
+func TestDefaultIngressClass(t *testing.T) {
+	name := controller.IngressClassName(manifests.DefaultIngressControllerName)
+	ingressclass := &networkingv1.IngressClass{}
+	if err := kclient.Get(context.TODO(), name, ingressclass); err != nil {
```

Comment:
How about we spice this test up a bit by checking that the default ingress controller's ingress class has the default annotation?